src/main/java/com/android/volley/toolbox/AsyncHttpStack.java

```diff
@@ -57,14 +57,14 @@ public abstract void executeRequest(
      * This method sets the non blocking executor to be used by the stack for non-blocking tasks.
      * This method must be called before executing any requests.
      */
-    @RestrictTo({RestrictTo.Scope.SUBCLASSES, RestrictTo.Scope.LIBRARY_GROUP})
+    @RestrictTo({RestrictTo.Scope.LIBRARY_GROUP})
```

Comment:
The problem here is that it prevents someone outside the Volley from extending AsyncHttpStack or AsyncNetwork, which is totally valid. Sounds like you're still looking into why this is happening, but let me know if I can help.

Reply:
One thing I was unclear on: does implementing these methods directly in AsyncNetwork/AsyncHttpStack solve this issue? Like, would we not need the RestrictTo.Scopt.SUBCLASSES part, since we're doing that?